### ISSUE TITLE
Unable to access the request before visiting a page

### ISSUE THREAD

**Issue by pjedrzejewski** [FriendsOfBehat/MinkDebugExtension#10]
Seems like some edge-case is not covered and is muting the real error for me.

```
bin/behat --strict -f progress -p cached -s addressing -v

  [Behat\Mink\Exception\DriverException]
  Unable to access the request before visiting a page

Exception trace:
 () at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/mink-browserkit-driver/src/BrowserKitDriver.php:156
 Behat\Mink\Driver\BrowserKitDriver->getCurrentUrl() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/mink/src/Session.php:239
 Behat\Mink\Session->getCurrentUrl() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/lakion/mink-debug-extension/src/Listener/FailedStepListener.php:100
 Lakion\Behat\MinkDebugExtension\Listener\FailedStepListener->logPageContent() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/lakion/mink-debug-extension/src/Listener/FailedStepListener.php:90
 Lakion\Behat\MinkDebugExtension\Listener\FailedStepListener->logFailedStepInformations() at n/a:n/a
 call_user_func() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:158
 Symfony\Component\EventDispatcher\EventDispatcher->doDispatch() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Testwork/EventDispatcher/TestworkEventDispatcher.php:39
 Behat\Testwork\EventDispatcher\TestworkEventDispatcher->dispatch() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Behat/EventDispatcher/Tester/EventDispatchingStepTester.php:87
 Behat\Behat\EventDispatcher\Tester\EventDispatchingStepTester->tearDown() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Behat/Tester/StepContainerTester.php:62
 Behat\Behat\Tester\StepContainerTester->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Behat/Tester/Runtime/RuntimeScenarioTester.php:76
 Behat\Behat\Tester\Runtime\RuntimeScenarioTester->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Behat/Hook/Tester/HookableScenarioTester.php:74
 Behat\Behat\Hook\Tester\HookableScenarioTester->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Behat/EventDispatcher/Tester/EventDispatchingScenarioTester.php:103
 Behat\Behat\EventDispatcher\Tester\EventDispatchingScenarioTester->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Behat/Tester/Runtime/RuntimeFeatureTester.php:83
 Behat\Behat\Tester\Runtime\RuntimeFeatureTester->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Behat/Hook/Tester/HookableFeatureTester.php:72
 Behat\Behat\Hook\Tester\HookableFeatureTester->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Behat/EventDispatcher/Tester/EventDispatchingFeatureTester.php:71
 Behat\Behat\EventDispatcher\Tester\EventDispatchingFeatureTester->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Testwork/Tester/Runtime/RuntimeSuiteTester.php:63
 Behat\Testwork\Tester\Runtime\RuntimeSuiteTester->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Testwork/Hook/Tester/HookableSuiteTester.php:73
 Behat\Testwork\Hook\Tester\HookableSuiteTester->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Testwork/EventDispatcher/Tester/EventDispatchingSuiteTester.php:72
 Behat\Testwork\EventDispatcher\Tester\EventDispatchingSuiteTester->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Testwork/Tester/Runtime/RuntimeExercise.php:71
 Behat\Testwork\Tester\Runtime\RuntimeExercise->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Testwork/EventDispatcher/Tester/EventDispatchingExercise.php:70
 Behat\Testwork\EventDispatcher\Tester\EventDispatchingExercise->test() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Testwork/Tester/Cli/ExerciseController.php:146
 Behat\Testwork\Tester\Cli\ExerciseController->testSpecifications() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Testwork/Tester/Cli/ExerciseController.php:108
 Behat\Testwork\Tester\Cli\ExerciseController->execute() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Testwork/Cli/Command.php:63
 Behat\Testwork\Cli\Command->execute() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:256
 Symfony\Component\Console\Command\Command->run() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:838
 Symfony\Component\Console\Application->doRunCommand() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:189
 Symfony\Component\Console\Application->doRun() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/src/Behat/Testwork/Cli/Application.php:102
 Behat\Testwork\Cli\Application->doRun() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:120
 Symfony\Component\Console\Application->run() at /Users/pjedrzejewski/Sites/sylius.dev/vendor/behat/behat/bin/behat:31

behat [-s|--suite SUITE] [-f|--format FORMAT] [-o|--out OUT] [--format-settings FORMAT-SETTINGS] [--init] [--lang LANG] [--name NAME] [--tags TAGS] [--role ROLE] [--story-syntax] [-d|--definitions DEFINITIONS] [--append-snippets] [--no-snippets] [--strict] [--rerun] [--stop-on-failure] [--dry-run] [--] [<paths>]
```

**Comment by pjedrzejewski** [comment]
Note: When you use a different formatter (other than progress) you can actually see the real exception.

**Comment by pamil** [comment]
Should be fixed by #13, thanks for reporting! :+1:
